src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Azure.WebJobs.Script.Properties;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics.Extensions
+{
+    internal static class LoggerExtension
```

Comment:
There's a bunch of new logger extensions here, loosely tied to feature areas. However, this one is just LoggerExtension and seems a dumping ground for a bunch of misc logs?

What's the right pattern - should we have just one big extension for all?

Reply:
When we chatted with @fabiocav  we decided to split logging. I agree loggers are loosely tied to feature areas. My idea was to create a new extension if we have more then 10 logs in a class. We can split by log category. Also one big extension looks good to me.